source/application.py

```diff
@@ -8,12 +8,12 @@
 
 DAO = Database()
 DAO.add_device(SimpleLock())
-DAO.add_device(DimmableLight())
+#DAO.add_device(DimmableLight())
```

Comment:
We have a real dimmable light now so I removed the mock one. The class still exists